### PR TITLE
Refactor: make it possible to add own paths

### PR DESCRIPTION
**Pull request by nichtsfrei**
To set your own paths instead of the defaults you have to follow the
instructions within `Use own paths instead of defaults` within the
README.md

### CODE REVIEW

**Comment by Kraemii** [comment]
Wouldn't it be more convenient to just use a single environment variable like INSTALL_PREFIX, which is already used when installing the stack within a defined directory?

**Comment by Kraemii** [comment]
My idea for this would be setting environment variables for the yaml file within the makefile for the openvas-data, notus-data and scan-configs respectively. These can then be used withing the yaml files. To use the yaml file without the makefile you can use a sysntax like `${VAR:default_value}` to also support default values. Not sure if this will work though.

**Comment by nichtsfrei** [comment]
This yaml template alla `${VAR:default}` does not work, hence this workaround via make.

Regarding the INSTALL_PREFIX, maybe, I don't use that but it would endup in 4 variables:
- INSTALL_PREFX
- each already available variables based on `INSTALL_PREFIX` instead of full path.

**Comment by nichtsfrei** [comment]
Added INSTALL_PREFIX

File: Makefile [hunk 1]
```diff
@@ -1,44 +1,79 @@
...
+endif
+
+NASL_TARGET_DEFAULT := ${INSTALL_PREFIX}/lib/openvas/plugins
+nasl_target := ${NASL_TARGET_DEFAULT}
+
+NOTUS_TARGET_DEFAULT := ${INSTALL_PREFIX}/lib/notus
+notus_target := ${NOTUS_TARGET_DEFAULT}
+
+SC_TARGET_DEFAULT := ${INSTALL_PREFIX}/lib/gvm/data-objects/gvmd/22.04/scan-configs
+sc_target := ${SC_TARGET_DEFAULT}
```
Comment (Kraemii) [inline Makefile:16]:
```suggestion
ifndef INSTALL_PREFIX
	INSTALL_PREFIX := 
endif

NASL_TARGET_DEFAULT := ${INSTALL_PREFIX}/var/lib/openvas/plugins
nasl_target := ${NASL_TARGET_DEFAULT}

NOTUS_TARGET_DEFAULT := ${INSTALL_PREFIX}/var/lib/notus
notus_target := ${NOTUS_TARGET_DEFAULT}

SC_TARGET_DEFAULT := ${INSTALL_PREFIX}/var/lib/gvm/data-objects/gvmd/22.04/scan-configs
sc_target := ${SC_TARGET_DEFAULT}
```
The `INSTALL_PREFIX` does not contain `/var`

File: README.md [hunk 2]
```diff
@@ -92,6 +92,30 @@ kubectl scale deployments/slsw --replicas=100
...
 
+
+### Use own paths instead of defaults
+
+If you follow the standard setup
+
+- $YOUR_PATH/lib/openvas/plugins
+- $YOUR_PATH/lib/notus
+- $YOUR_PATH/lib/gvm/data-objects/gvmd/22.04/scan-configs
+
```
Comment (Kraemii) [inline README.md:103]:
```suggestion
- $YOUR_PATH/var/lib/openvas/plugins
- $YOUR_PATH/var/lib/notus
- $YOUR_PATH/var/lib/gvm/data-objects/gvmd/22.04/scan-configs

```

**Pull request merged**